assembl/static2/js/app/hoc/aux/aux.jsx

```diff
@@ -0,0 +1,17 @@
+/*
+  `aux` component is a high order component used to
```

Comment:
we don't need since `React.Fragment` was introduced in React 16 : https://reactjs.org/docs/fragments.html

Reply:
True that @cedricmessiant it was just an example to show the team what kind of component Fragment is. But definitely, we should use Fragment.